packages/router/__tests__/browser-test.ts

```diff
@@ -1,7 +1,3 @@
-/**
- * @jest-environment ./__tests__/custom-environment.js
```

Comment:
Unsure why we had 2 ways of doing this but I moved the custom environment stuff into `setup.ts`